### PR TITLE
Added support for character count

### LINKED ISSUE #114: Add support for character count
- Add support to show the number of characters entered.

@AbhayVAshokan _a 
enhancement _t

### PR DESCRIPTION
**Pull request by AbhayVAshokan**
Fixes #114 
- Installed `@tiptap/extension-character-count` tiptap extension.
- Added support for character count and word count.
- Added documentation.

Demo: https://vimeo.com/660427999/19e1126ada

@karthiknmenon _a could you please look at the new changes and verify if it matches with our requirements?
cc: @goutham-subramanyam

### CODE REVIEW

**Comment by karthiknmenon** [comment]
@AbhayVAshokan Feature LGTM. 
@labeebklatif _a Could you please review the code and assign it to Goutham for a UI review? 

cc: @goutham-subramanyam, @amaldinesh7

File: lib/styles/editor/_character-count.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.neeto-editor-character-count {
```
Comment (labeebklatif) [inline lib/styles/editor/_character-count.scss:1]: Can we have these styles inside the editor.scss itself? This is a tiny element which doesn't seem to need a stylesheet of it's own

File: lib/components/Editor/index.js [hunk 2]
```diff
@@ -136,6 +138,14 @@ const Tiptap = (
       ) : (
         <EditorContent editor={editor} {...otherProps} />
       )}
+      <p className="neeto-editor-character-count">
+        {characterLimit
+          ? `${
+              characterLimit - editor?.storage?.characterCount?.characters()
+            } characters remaining`
```
Comment (labeebklatif) [inline lib/components/Editor/index.js:145]: Can we extract this `p` tag and content to separate component which may take `characterLimit` and `editor` as props. The index file is getting to be too lengthy and messy

**Review by labeebklatif — CHANGES_REQUESTED** [review]
@AbhayVAshokan _a Few minor suggestions. Please check

**Comment by AbhayVAshokan** [comment]
@labeebklatif _a please review the new changes.
- Extracted the character count logic to a separate component: `CharacterCount` (consistent with name of extension).
- Removed `_character-count.scss` stylesheet, add the class along with `editor.scss`.

**Review by labeebklatif — APPROVED** [review]
LGTM!

**Pull request merged**